autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/TabularColumnTransformer.py

```diff
@@ -57,9 +57,9 @@ def fit(self, X: Dict[str, Any], y: Any = None) -> "TabularColumnTransformer":
         if len(X['dataset_properties']['categorical_columns']):
             categorical_pipeline = make_pipeline(*preprocessors['categorical'])
 
-        self.preprocessor = make_column_transformer(
-            (numerical_pipeline, X['dataset_properties']['numerical_columns']),
-            (categorical_pipeline, X['dataset_properties']['categorical_columns']),
+        self.preprocessor = ColumnTransformer([
+            ('numerical_pipeline', numerical_pipeline, X['dataset_properties']['numerical_columns']),
```

Comment:
If there are only categorical columns, numerical_pipeline is set to drop. Does it make sense?

Can we enhance [this](https://github.com/automl/Auto-PyTorch/blob/8783240e2ef818d4cbb3447ae89a1319e14408c4/test/test_pipeline/components/preprocessing/test_tabular_column_transformer.py#L20) check to make sure all columns are available after the transformer?

Reply:
I think it makes sense as all the columns would be categorical and they would be passed through the categorical pipeline and drop would not do anything. I'll add the test to make sure all the columns are passed